components/game/game.js

```diff
@@ -1,25 +1,79 @@
 import gameStyles from './game.module.scss';
-import { useState } from 'react';
+import styles from "@/app/page.module.css";
+import { useState, useEffect } from 'react';
 import Hand from "@/components/gameComponents/hand.js";
-import { dictionaryCombinations } from '@/components/utilities/combination.js';
+import { dictionaryCombinations, highestValue } from '@/components/utilities/combination.js';
+import { mapCard, icons } from '../utilities/card';
 
 const Game = () => {
+  const [shuffledDeck, setDeck] = useState([]);
   const [deckIsShuffled, shuffleDeck] = useState(false);
   const [introIsVisible, showIntro] = useState(false);
   const [playerTurn, setPlayerTurn] = useState(0);
   const [hands, setHands] = useState(null);
   const [comboIsValid, setComboStatus] = useState(null);
   const [currentTurnCombo, setCurrentTurnCombo] = useState('single');
-
+  const [previousPlayedCombo, setPreviousPlayedCombo] = useState([]);
   const [selectCombo, setComboSelect] = useState('single');
 
+  const [endCycleClause, setEndCycleClause] = useState(false);
+
   // Build Card Deck
   const suites = ['spades', 'clubs', 'diamonds', 'hearts'];
   // 3-10 are normal cards, 11 is Jack, 12 is Queen, 13 is King, 14 is Ace, 15 is 2.
-  const numbers = ['3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15'];
+  const numbers = [3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
   let value = 1; // 3 of spades is the lowest card
   const deck = numbers.map((number, idx) => suites.map((suite, i) => ({ number, suite, value: value++ }))).flat();
 
+  useEffect(() => {
+    setDeck(shuffle(deck));
+  }, []);
+
+  useEffect(() => {
+    if (!hands) {
+      return;
+    }
+     /**
+      * @description End logic if all hands have passed
+      * NOTE: This logic is not complete and does not cater to all use cases.
+      * NOTE: THIS IS STRICTLY FOR TESTING
+      */
+
+    let checkEndCycle = hands.filter((hand) => hand.skipped !== true);
```

Comment:
maybe we should define our linter more explicitly, but i'm more used to seeing this format

```suggestion
    let checkEndCycle = hands.filter(hand => hand.skipped !== true);
```

Reply:
We can go with that.